### DISCUSSION TITLE
Error when adding a TAG

### DISCUSSION THREAD

**Discussion by Tanote650** [open-webui/open-webui#3776] [General]
# Bug Report

## Description

**Bug Summary:**
An error occurs when adding TAGs in the chat. The TAG is multiplied in the view.

**Steps to Reproduce:**
Simply add several TAGs to a chat.

## Environment

- **Open WebUI Version:** v0.3.8 
- **Browser (if applicable):** [Safari]

**Comment by ProjectMoon** [comment]
I noticed this too. The bug report is maybe not detailed, but it is happening. It happens with tags that are newly created after the upgrade, as well as tags that existed before the upgrade. When you start adding the same tag to multiple conversations, the tag itself gets multiplied in the view.
![image](https://github.com/open-webui/open-webui/assets/183856/42dcb367-c2fc-433f-921f-d177b5feae8f)

**Comment by jonnywright** [comment]
I am also experiencing this, but just to be clear on steps to reproduce, this can happen by just adding one tag (if that tags is one which has re-produced itself)
